### ISSUE TITLE
Include hasMany with fixtures does not seem to work

### ISSUE THREAD

**Issue by topaxi** [samselikoff/ember-cli-mirage#567]
I have something similar to:

`mirage/models/user.js`:
```js
import { Model, hasMany } from 'ember-cli-mirage'

export default Model.extend({
  comments: hasMany('text')
})
```

`mirage/models/text.js`:
```js
import { Model, belongsTo } from 'ember-cli-mirage'

export default Model.extend({
  author: belongsTo('user')
})
```

`mirage/fixtures/user.js`:
```js
export default [
  { id: 1, username: 'topaxi' }
]
```

`mirage/fixtures/texts.js`:
```js
export default [
  { id: 1, content: 'foo', authorId: 1 },
  { id: 2, content: 'bar', authorId: 1 },
  { id: 3, content: 'baz', authorId: 1 }
]
```

`mirage/scenarios/default.js`:
```js
export default (server) => {
  server.loadFixtures('users')
  server.loadFixtures('texts')
}
```

`mirage/config.js`:
```js
export default () => {
  this.namespace = 'api/v1'

  this.get('/v1/users')
}
```

Now if I'm requesting `/api/v1/users/1?include=comments`, I will only get the user without any comments/texts included.
I'm not sure if I'm doing something wrong.
Should I specify the one-to-many relationship only once?
Do the one-to-many includes only work if I create the models / objects by hand?

**Comment by samselikoff** [comment]
I think your setup is correct. What version are you on? [This commit](2400b33a1e6bd9371436875b684bc6ae943f6b28) had some fixes for named relationships, so if you're not on that I'd say try that first with what you have.

**Comment by topaxi** [comment]
I'm currently using the master branch. If this should work, I'll try to create a minimal app to reproduce this as there are a lot of other models involved in my current app.

**Comment by samselikoff** [comment]
Minimal app would be very great, thank you

**Comment by topaxi** [comment]
Here is the minimal app to reproduce: https://github.com/topaxi/mirage-include-hasmany

The example doesn't output anything, you have to check the console :)

I tried with `/api/v1/users?include=comments` or `/api/v1/users/1?include=comments`, but neither of them seem to work for me.

**Comment by samselikoff** [comment]
This is sort of a bug, sorry about the trouble it caused. It's because `hasMany` tries to ensure there's a foreign key on the child model, if it doesn't exist. Because the fk is a named foreign key in this case, your child model ends up with two fks.

The solution is to name the inverse. Add this to your user model:

```
comments: hasMany('text', {inverseOf: 'author'})
```

and everything should work. I'll open a new issue to improve this in the future. Thanks for reporting this!

**Comment by topaxi** [comment]
It now works with `ember-cli-mirage#master`, thanks!
